plutus-book/doc/02-glossary.adoc

```diff
@@ -0,0 +1,31 @@
+= Glossary
```

Comment:
There's a glossary style you can apply: https://asciidoctor.org/docs/user-manual/#user-glossary

Not sure it does much, but might as well.